frontend/packages/metal3-plugin/src/selectors/machine.ts

```diff
@@ -0,0 +1,14 @@
+import { apiVersionForModel, MachineKind } from '@console/internal/module/k8s';
+import { compareOwnerReference } from '@console/kubevirt-plugin/src/utils';
+import { MachineSetModel } from '@console/internal/models';
+import { getOwnerReferences } from '@console/shared/src';
+
+export const getMachineMachineSetOwner = (machine: MachineKind) => {
+  const desiredReference = {
+    apiVersion: apiVersionForModel(MachineSetModel),
+    kind: MachineSetModel.kind,
+  } as any;
+  return (getOwnerReferences(machine) || []).find((reference) =>
+    compareOwnerReference(desiredReference, reference, true),
```

Comment:
`@console/metal3-plugin` nor `@console/shared` should import from '@console/kubevirt-plugin'

Reply:
you are right: https://github.com/openshift/console/pull/4014